index.js

```diff
@@ -201,8 +201,9 @@ fastifyOauth2.LINKEDIN_CONFIGURATION = {
 fastifyOauth2.GOOGLE_CONFIGURATION = {
   authorizeHost: 'https://accounts.google.com',
   authorizePath: '/o/oauth2/v2/auth',
-  tokenHost: 'https://www.googleapis.com',
-  tokenPath: '/oauth2/v4/token'
+  tokenHost: 'https://oauth2.googleapis.com',
```

Comment:
I would not change it - the official doc has not changed as well:
https://developers.google.com/identity/protocols/oauth2?hl=en

That seems more an insider URL that may change overtime, `googleapis.com` seems a proxy that will work forever

Reply:
Yes, this may be true for some apis, however when I tried to revoke a token using the revoke method, I got an error.

<img width="478" alt="Screenshot 2023-05-28 at 12 41 21" src="https://github.com/fastify/fastify-oauth2/assets/18070500/f5bf8146-27d6-411b-b0b6-35dadce5e0e0">

I noticed the api has changed when I searched for revoke tokens api documentation:
https://developers.google.com/identity/protocols/oauth2/web-server?hl=en#tokenrevoke

Reply:
@myksao is actually right here.

Google's `https://accounts.google.com` is a OIDC Provider which supports metadata discovery.
Read more on discovery endpoint [standard here](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata).

So, discovery endpoint (when present) MUST be authoritative reference for other exposed OP's (OIDC Provider) endpoints. 
(Not documentation, since we all know how docs never get enough love they deserve 😢 )
Discovery endpoint for Google's OP, is `https://accounts.google.com/.well-known/openid-configuration`

```js
const { Issuer } = require('openid-client')
const GOOGLE_ISSUER = 'https://accounts.google.com'

Issuer.discover(GOOGLE_ISSUER)
  .then(({
    metadata: {
      authorization_endpoint,
      token_endpoint,
      revocation_endpoint,
      // ...other metadata
    }
  }) => {
    const endpoints = [authorization_endpoint, token_endpoint, revocation_endpoint]
    const [
      { host: authorizeHost, path: authorizePath },
      { host: tokenHost, path: tokenPath },
      { path: revokePath }
    ] = endpoints.map(ep => new URL(ep)).map(url => ({ host: `${url.protocol}//${url.host}`, path: url.pathname }))

    const GOOGLE_CONFIGURATION = { authorizeHost, authorizePath, tokenHost, tokenPath, revokePath }
    console.log(GOOGLE_CONFIGURATION)
  })
=> {
 authorizeHost: 'https://accounts.google.com',
 authorizePath: '/o/oauth2/v2/auth',
 tokenHost: 'https://oauth2.googleapis.com',
 tokenPath: '/token',
 revokePath: '/revoke'
}
```